internal/proxyservice/tcp/mssql/connector.go

```diff
@@ -0,0 +1,21 @@
+package mssql
+
+import (
+	"net"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+)
+
```

Comment:
can we add some documentation to this file? what is its purpose? maybe also link to an existing connector